app/views/course/assessment/submission/submissions/edit.html.slim

```diff
@@ -9,6 +9,8 @@
     h4 = t('.files')
     = render partial: 'layouts/materials', locals: { folder: @assessment.folder }
 
+= render 'progress' if can?(:grade, @submission)
```

Comment:
submission can only be `:grade` after submitted, attempting state still won't display progress.